packages/go_router/doc/logging.md

```diff
@@ -0,0 +1,8 @@
+To enable log output, enable the `debugLogDiagnostics` parameter:
```

Comment:
Should we move this to the Configuration topic?

Reply:
SGTM, I'll remove this topic.